gradle/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.3.0'
+	id 'io.spring.dependency-management' version '1.1.5'
+}
+
+group = 'io.paketo'
```

Comment:
chang group to "com.example" as the other test case

Reply:
Updated.